other/webpack.config.es6.js

```diff
@@ -185,7 +184,7 @@ const config = {
     formatter: require('eslint-friendly-formatter'),
   },
 
-  devtool: 'source-map',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
I think we only want this setting for `ON_DEV`. (And maybe `ON_TEST`.)

I tried running `npm run build` with these changes and it looks like it's actually putting all the prod code in `eval()` statements. It's also including the source mappings inline.

The downside to keeping `source-map` for production is that the one-time build takes quite a bit longer, but the better performance and security of the code is worth the wait.